resources/definitions/alya3dp.def.json

```diff
@@ -1,50 +1,50 @@
+        "preferred_material": "generic_pla",
+        "has_machine_quality": true,
+        "has_materials": true,
+        "has_variants": false,
+        "supports_usb_connection": false,
         "machine_extruder_trains":
         {
             "0": "alya3dp_extruder_0"
         }
     },
 
     "overrides": {
-        "machine_width": {
-            "default_value": 100
-        },
-        "machine_height": {
-            "default_value": 133
-        },
-        "machine_depth": {
-            "default_value": 100
-        },
-        "machine_center_is_zero": {
-            "default_value": false
-        },
-        "machine_head_polygon": {
-            "default_value": [
-                [75, 18],
-                [18, 18],
-                [18, 35],
-                [75, 35]
-            ]
-        },
-        "gantry_height": {
-            "default_value": 55
-        },
-        "machine_gcode_flavor": {
-            "default_value": "RepRap"
-        },
-        "machine_start_gcode": {
-            "default_value": ";Sliced at: {day} {date} {time}\n;Basic settings: Layer height: {layer_height} Walls: {wall_thickness} Fill: {fill_density}\n;Print time: {print_time}\n;Filament used: {filament_amount}m {filament_weight}g\n;Filament cost: {filament_cost}\n;M190 S{print_bed_temperature} ;Uncomment to add your own bed temperature line\n;M109 S{print_temperature} ;Uncomment to add your own temperature line\nG21        ;metric values\nG90        ;absolute positioning\nM82        ;set extruder to absolute mode\nM107       ;start with the fan off\nG28 X0 Y0  ;move X/Y to min endstops\nG28 Z0     ;move Z to max endstops\nG1 Z115.0 F{speed_travel} ;move th e platform up 20mm\nG28 Z0     ;move Z to max endstop\nG1 Z15.0 F{speed_travel} ;move the platform down 15mm\nG92 E0                  ;zero the extruded length\nG1 F200 E3              ;extrude 3mm of feed stock\nG92 E0                  ;zero the extruded length again\nG1 F{speed_travel}\nM301 H1 P26.38 I2.57 D67.78\n;Put printing message on LCD screen\nM117 Printing..."
+        "machine_name": { "default_value": "ALYA 3DP" },
+        "machine_heated_bed":          {   "default_value": false            },
+        "machine_width":               {   "default_value": 100             },
+        "machine_height":              {   "default_value": 133             },
+        "machine_depth":               {   "default_value": 100             },
+        "machine_center_is_zero":      {   "default_value": false           },
+        "gantry_height":               {   "default_value": 55             },
+        "retraction_amount":           {   "default_value": 1.5 },
+        "support_enable":              {   "default_value": true},
+        "machine_head_with_fans_polygon": {
+            "default_value": [[75, 18],[18, 18],[18, 35],[75, 35]]
+        },
+        "adhesion_type": {"options": {"raft": "Raft" ,"none": "None", "brim": "Brim"},  "default_value": "raft"},
```

Comment:
Skirt is not allowed, and neither are any future adhesion types? Why are these options adjusted?

Reply:
Well, we don't have a heated bed, and we have a flexible print surface with holes. It allows easy printing using a raft. But without raft, prints come out quite ugly. If you don't mind, we'd like to leave it that way. If you insist, i could of course just use a default value as raft-